liquidctl/cli.py

```diff
@@ -45,6 +46,10 @@
   --version                      Display the version number
   --help                         Show this message
 
+Config file options:
+  -f, --file <file>              Select a specific config file to use
+  --ignore-config                Do not use any of the config files
```

Comment:
`--ignore-config` is probably unnecessary, `--file /dev/null` or `--file <some empty file>` are already sufficient for this very niche use case.

Reply:
ohhh I like the `/dev/null` idea, but Ill need to make sure that the parser actually handles that

Reply:
The parser would only need to handle an empty configuration file... the kernel should handle the rest for us : )